python/chemiscope/sphinx/utils.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+
+
+def copy_file(src_file, dst_file):
+    """
+    Copy a single file from source to destination
+
+    Parameters:
+    - src_file (str): The path to the source file
+    - dst_file (str): The path to the destination file
+    """
+    dst_dir = os.path.dirname(dst_file)
+    os.makedirs(dst_dir, exist_ok=True)
+    shutil.copyfile(src_file, dst_file)
+
+
+def copy_static_folder(app, exception):
+    """
+    Copy static files required for the chemiscope widget
+
+    Parameters:
+    - app (Sphinx.application.Sphinx): The sphinx application object
+    - exception (Exception): The exception that occurred during Sphinx build, if any
+    """
+    if exception:
+        return
+    current_file_dir = os.path.dirname(__file__)
+    static_dir = os.path.join(current_file_dir, "static")
+    build_static_dir = os.path.join(app.outdir, "static")
```

Comment:
I think there is an `html_static_dir` setting in sphinx (or with a similar name) that we should use here

Reply:
But I'm wondering that user can specify any folders here but in out html we rely exactly on the "static". Do you think it is worth passing static folder name and replacing it in the html template?

Reply:
Ok, so I've been checking the docs for `html_static_path` https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_static_path, and it is actually a list of path that will be copied in full in the build output. It looks like the files are copied to `_static`, and this output path is not user-configurable.

Maybe a better solution here is to add our own static path (`chemiscope/sphinx/static`) to this variable (in the extension's `setup`); and load from sphinx's `_static` directory in the output (i.e. load from `/_static/js/chemiscope.min.js`). Does this make sense?